dataclients/kube/definitions.go

```diff
@@ -0,0 +1,115 @@
+package kube
```

Comment:
for the other case, with the address option, i agree and changing right away. For the package name, i am a bit hesitant, but without a clear rational reasoning. Anyone with additional opinion on this.